devel/libmsp430/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libmsp430
+version             3.15.1.1
+license             BDS-3
+
+categories          devel
+platforms           darwin
+
+maintainers         {@edilmedeiros gmail.com:jose.edil+macports} \
+                    openmaintainer
+
+description         Low-level MSP430 USB drivers
+long_description    These drivers provide an interface between the host \
+                    system MSP Debug Stack library (libmsp430.dylib) and \
+                    the FET’s USB interface. This is accomplished by using \
+                    a Communication Device Class (CDC) or Virtual COM Port \
+                    (VCP) protocol.
+
+homepage            https://www.ti.com/tool/MSPDS
+master_sites        https://dr-download.ti.com/software-development/driver-or-library/MD-4vnqcP1Wk4/${version}/
+
+distfiles           MSPDebugStack_OS_Package_3_15_1_1.zip
+checksums           md5     c16ee393e6d5388e8352aed6a716b7ba \
+                    rmd160  efdad29b4f2247d92adbbe0ebc44a8e37140ca7d \
+                    sha256  e3a59a98c43de7a92e5814d8c3304026165e6d2551e60acaca1f08c6b1a4bac8 \
+                    size    2184052
+
+depends_lib         port:boost178 \
```

Comment:
Stupid question: would it work with a newer Boost? (Not necessary mind you, just asking.)

Reply:
Probably will work, but I had already that version running for quite a long time.
Is there a way to say "from version 1.78" instead of "exactly version 1.78"?

Reply:
Or "any version of boost"?

Reply:
I don't know how to do that, maybe ask on the macports devel list?